src/main/java/de/retest/recheck/ignore/SearchFilterFiles.java

```diff
@@ -82,6 +82,14 @@ private SearchFilterFiles() {}
 				.orElse( Collections.emptyList() );
 	}
 
+	public static List<Pair<String, FilterLoader>> getUserFilterFiles() {
+		final Path userFilterFolder = Paths.get( System.getProperty( "user.home" ), ".retest", "filter" );
```

Comment:
We probably want several constants here:

* Have something for the user retest folder, maybe in `Properties`? We might also want to reuse `.retest` from `ProjectConfiguration#RETEST_PROJECT_CONFIG_FOLDER` and/or move the string to somewhere else?
* Reuse `BASIC_FILTER_DIR` and/or `WEB_FILTER_DIR` as we probably want the same basic structure within the `.retest` folder, regardless of where it is (project, user, …).